web/views/user_view.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Cog.V1.UserView do
+  use Cog.Web, :view
+
+  def render("user.json", %{user: user}) do
+    %{id: user.id,
+      username: user.username,
+      first_name: user.first_name,
+      last_name: user.last_name,
+      email_address: user.email_address,
+      group_memberships: render_groups(user.group_memberships)}
```

Comment:
Not really. It was more about consistency with what field is accessed here. That said, I can easily change it to groups, if that's preferred.